File: week-02/day-2/2023.04.11/untitled/src/GuessTheNumber.java
import java.util.Scanner;

public class GuessTheNumber {
    public static void main(String[] args) {

        int number = 4;

        System.out.print("Please enter your guess: ");
        Scanner scan = new Scanner(System.in);
        int input1 = scan.nextInt();
        do {
            if (input1 > number) {
                System.out.println("The stored number is smaller than " + input1);
            } else if (input1 < number) {
                System.out.println("The stored number is greater than " + input1);
            }
            System.out.println("Please enter your guess:");
            input1 = scan.nextInt();

        }while (input1 != number);

    }
}

File: week-07/day-1/Workshop 15.5.2023/guess_the_number.py
# Write a program that stores a number and the user has to figure it out.
# The user can input guesses. After each guess the program responds
# with a message indicating whether the stored number is smaller or
# greater than the guess.
# The program ends (exits) when the user finds the stored number
#
# Example:
#
# Please enter your guess: 3
# The stored number is greater than 3
# Please enter your guess: 10
# The stored number is smaller than 10
# Please enter your guess: 8
# You have found the stored number 8
#
stored_number = 8
inp = input("Please enter your guess: ")
while int(inp) != stored_number:
    if int(inp) < stored_number:
     print("The stored number is greater than " + inp)
    elif int(inp) > stored_number:
        print("The stored number is smaller than " + inp)
    inp = input("Please enter your guess: ")

print("You have found the stored number " + inp)